guides/v2.3/graphql/reference/paypal-create-payflow-pro-token.md

```diff
@@ -69,13 +69,17 @@ Attribute |  Data Type | Description
 
 ### PayflowProUrlInput object
 
+The `PayflowProUrlInput` object contains a set of relative URLs that PayPal will use in response to various actions during the authorization process. Magento prepends the base URL to this value to create a full URL. For example, if the full URL is https://www.example.com/path/to/page.html, the relative URL is path/to/page.html. 
```

Comment:
should you put the urls in backticks like in the attribute table above and below?